react-accordion/my-app/src/Accordion.tsx

```diff
@@ -0,0 +1,33 @@
+import { TopicCard } from './TopicCard';
+import { useState } from 'react';
+
+type Topics = {
```

Comment:
This is actually just one single topic, so it should be singular.

You probably should export this type so you can use it in your other components, such as TopicCard.